web/docs/integrations/suricata.md

```diff
@@ -0,0 +1,44 @@
+# Suricata
```

Comment:
@dominiklohmann we could add a section at the end how on simulating Suricata with Speeve, like from your demo:

```bash
~/go/bin/speeve spew --persec 100 --profile <SPEEVE_PROFILE_PATH> | docker compose run --no-TTY  tenzir 'remote save file -a /tmp/eve.sock'
```

WDYT?

Reply:
I think that is better suited for a package in the library rather than as part of this integration. I'd like to focus on the tool only.